frontend/javascripts/maintenance_banner.tsx

```diff
@@ -0,0 +1,106 @@
+import {
+  MaintenanceInfo,
+  listCurrentAndUpcomingMaintenances,
+  updateNovelUserExperienceInfos,
+} from "admin/admin_rest_api";
+import { Alert } from "antd";
+import FormattedDate from "components/formatted_date";
+import { useInterval } from "libs/react_helpers";
+import { setActiveUserAction } from "oxalis/model/actions/user_actions";
+import { Store } from "oxalis/singletons";
+import React, { useState } from "react";
+
+export function MaintenanceBanner() {
+  const { activeUser } = Store.getState();
+  const [currentAndUpcomingMaintenances, setcurrentAndUpcomingMaintenances] = useState<
+    Array<MaintenanceInfo>
+  >([]);
+  const [position, setPosition] = useState<Object>({ top: 48 });
+  const [isTop, setIsTop] = useState(true);
+  useInterval(
+    async () => {
+      if (activeUser == null) return;
+      setcurrentAndUpcomingMaintenances(await listCurrentAndUpcomingMaintenances());
+    },
+    10000, //TODO find value for production
```

Comment:
I think, once a minute would be okay. should be extracted into a top-level constant then.